tf/mod_source_parse.go

```diff
@@ -110,6 +120,7 @@ func ParseSource(modsource string) (Source, error) {
 
 		// We don't want : on the path. So we replace the possible :
 		// that can exist on the host.
+		subdir := parseURLSubdir(u)
 		path := filepath.Join(strings.Replace(u.Host, ":", "-", -1), u.Path)
 		path = strings.TrimSuffix(path, ".git")
```

Comment:
```suggestion
		// We don't want : on the path. So we replace the possible :
		// that can exist on the host.
		path := filepath.Join(strings.Replace(u.Host, ":", "-", -1), u.Path)
		path = strings.TrimSuffix(path, ".git")
		subdir := parseURLSubdir(u)
```
so the comment is close to the code it refers to.